README.md

```diff
@@ -125,7 +125,7 @@ The most frequently used `appmap-agent-js` parameters are:
 - `--command="_start command_"` : alternate method of specifying the app- or tests-starting command, wrapped in quotes
 - `--log-level=[debug|info|warning|error]` :  defaults to `info`
 - `--log-file=_file_` : location of log file, defaults to `stderr` 
-- `--output-dir=_directory_` : location of recorded AppMap files, default is `tmp/appmap` or `tmp/appmap/mocha`
+- `--appmap-dir=_directory_` : location of recorded AppMap files, default is `tmp/appmap` or `tmp/appmap/mocha`
```

Comment:
So what the other agents do is add the framework name to the appmap_dir and put the AppMaps there. For example is appmap_dir is tmp/appmap and the framework is minitest, the AppMaps go to tmp/appmap/minitest. So the appmap_dir is really the base directory of AppMaps and the agent can feel free to make subdirectories. 

Remote recordings traditionally go in $appmap_dir/remote.